api/config/config.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/joho/godotenv"
+)
+
+var (
+	StringConnectionBD = ""
```

Comment:
Aqui o nome ficou misturado inglês com português, e tbm vc utilizou uma [notação húngara](https://pt.wikipedia.org/wiki/Nota%C3%A7%C3%A3o_h%C3%BAngara) que não é uma prática muito boa em Go. Que tal:
```suggestion
	DBConnectionURL = ""
```